src/main/features/offer/views/offer-confirmation.njk

```diff
@@ -0,0 +1,45 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, submitButton %}
+{% from "support.njk" import supportWidget %}
+{% set title = t('Money claim') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+        <div class="govuk-box-highlight">
+          <h1 class="bold-large">{{ t('Your offer has been sent to the %s',
+          {
+          postProcess: 'sprintf',
+          sprintf: [claim.claimData.claimant.name]
+          }
+          ) }}</h1>
+        </div>
+        <h2 class="heading-medium">{{ t('What happens next') }}</h2>
+        <p>
+        {{ t("%s can accept, reject or make a counter offer. We’ll let you know when they respond.",
+        {
+          postProcess: 'sprintf',
+          sprintf: [claim.claimData.claimant.name]
+        }
+        ) }}
+      </p>
+      <p>
+        {{ t('View your claim status on your <a href="%s">money claims account</a>.',
+          {
+            postProcess: 'sprintf',
+            sprintf: ['/dashboard']
+          }
+        ) | safe }}
+      </p>
+      
+      {{ supportWidget() }}
+
+      {{ t('<a href="%s" class="button">Finished</a>',
+        {
+            postProcess: 'sprintf',
+            sprintf: [responsePaths.confirmationPage.evaluateUri({ externalId: claim.externalId })]
```

Comment:
Is this the right link? I would have thought it would go to a survey

Reply:
as per prototype it remains on the same page

Reply:
The pattern we followed so far was to navigate the user to a survey after he finishes a journey. I would double-check with the UX team on this.